python/ray/rllib/evaluation/sampler.py

```diff
@@ -448,6 +455,13 @@ def _do_policy_eval(tf_sess, to_eval, policies, active_episodes):
         for k, v in pending_fetches.items():
             eval_results[k] = builder.get(v)
 
+    if clip_actions:
+        for policy_id, actions in eval_results.items():
+            policy = _get_or_raise(policies, policy_id)
+            actions, rnn_out_cols, pi_info_cols = eval_results[policy_id]
```

Comment:
```suggestion
            actions, rnn_out_cols, pi_info_cols = results
```

Reply:
(this is a little confusing to read otherwise, since the first `actions` is never used)